_includes/sharktank/3-up.html

```diff
@@ -27,15 +27,17 @@
                 </svg>                    
             </div>
             <div class="three-stories__content tablet-lg:grid-col-6">
-                <h2 class="margin-bottom-1">The Surgical Pause</h2>
-                <h3 class="margin-y-0">Reducing Post-surgery Mortality in Frail Veterans</h3>
-                <p class="font-body">The Surgical Pause utilizes the simple yet sophisticated Risk Analysis Index (RAI) to screen Veterans for frailty in 30 seconds, effectively flagging high-risk Veterans so that the surgical team can ensure that the proposed treatment plans both mitigate known risks and align with the Veterans' overarching life goals.</p>
-                <a href="https://marketplace.va.gov/innovations/preoperative-frailty-screening-prehabilitation" class="usa-button button-vha">More Information</a>
+                <h2 class="margin-bottom-1">Contraception on Demand </h2>
+                <h3 class="margin-y-0">Expanding access to contraceptive care for Veterans</h3>
+                <p class="font-body">Contraception on Demand trains pharmacy practitioners to conduct patient-centered 
+                  contraceptive counseling and to effectively prescribe hormonal contraception; improving Veteran 
+                  experiences with reproductive care at VA and expanding the scope of practice for VA clinical pharmacists.</p>
+                <a href="https://marketplace.va.gov/innovations/contraception-on-demand" class="usa-button button-vha">More Information</a>
             </div>
             <div class="tablet-lg:grid-col-5 tablet-lg:grid-offset-1 desktop-lg:grid-col-5 desktop-lg:grid-offset-1 widescreen:grid-col-5 widescreen:grid-offset-1 mobile:padding-top-5 desktop:padding-top-0">
                 <div class="three-stories__img">
                     <div class="three-stories__img-container">
-                        <a href="https://marketplace.va.gov/innovations/preoperative-frailty-screening-prehabilitation"><img src="../assets/img/sharktank/surgical-pause.jpg" class="img-fluid radius-lg" alt="Poster image of contestants and judges on Shark Tank Competition stage" /></a>
+                        <a href="https://marketplace.va.gov/innovations/contraception-on-demand"><img src="../assets/img/sharktank/contraception-on-demand.jpg" class="img-fluid radius-lg" alt="Image of people sitting around a table" /></a>
```

Comment:
Can remove "Image of" and just describe who is in the photo and what they are doing